conf/boards/chimera_1.0.makefile

```diff
@@ -0,0 +1,94 @@
+# Hey Emacs, this is a -*- makefile -*-
+#
+# chimera_1.0.makefile
+#
+# based on STM32F7
+# pre-version, only compatible with ChibiOS
+#
+
+BOARD=chimera
+BOARD_VERSION=1.0
+BOARD_DIR=$(BOARD)/chibios/v$(BOARD_VERSION)
+BOARD_CFG=\"boards/$(BOARD_DIR)/$(BOARD).h\"
+
+ARCH=chibios
+$(TARGET).ARCHDIR = $(ARCH)
+
+RTOS=chibios
+MCU=cortex-m7
+
+## FPU on F7
+USE_FPU=softfp
+USE_FPU_OPT= -mfpu=fpv5-d16
+
+USE_LTO=yes
+
+$(TARGET).CFLAGS += -DSTM32F7 -DPPRZLINK_ENABLE_FD -DUSE_HARD_FAULT_RECOVERY
+
+##############################################################################
+# Architecture or project specific options
+#
+# Define project name here (target)
+PROJECT = $(TARGET)
+
+# Project specific files and paths (see Makefile.chibios for details)
+CHIBIOS_BOARD_PLATFORM = STM32F7xx/platform.mk
+CHIBIOS_BOARD_PORT = ARMCMx/STM32F7xx/port.mk
+CHIBIOS_BOARD_LINKER = STM32F76xxI.ld
+CHIBIOS_BOARD_STARTUP = startup_stm32f7xx.mk
+ifeq ($(USE_ITCM),1)
```

Comment:
It is a fast RAM that is only accessible wen flashing with Black Magic Probe or with STLINK since dfu-util is not supporting it (can't access the needed address)